src/Database/Table/GroupedSelection.php

```diff
@@ -201,6 +201,10 @@ protected function loadRefCache()
 		}
 	}
 
+	protected function emptyResultSet($saveCache = TRUE, $deleteRererencedCache = TRUE) {
+		parent::emptyResultSet($saveCache, false);
```

Comment:
Sorry, my previous comment, which I deleted yesterday?, was wrong. I misread the code. The previous version was probably better :| I really sorry.